WalletWasabi/BranchNBound/BranchAndBound.cs

```diff
@@ -22,19 +22,18 @@ public BranchAndBound(List<Money> availableCoins)
 			SortedUTXOs = availableCoins.OrderByDescending(x => x.Satoshi).Select(c => c.Satoshi).ToArray();
 		}
 
-		public bool TryGetExactMatch(Money target, [NotNullWhen(true)] out List<Money>? selectedCoins)
+		public bool TryGetExactMatch(Money target, [NotNullWhen(true)] out List<Money> selectedCoins)
```

Comment:
The original version is more common in this code base. Not sure why it was changed.

Reply:
I tried to minimize the use of `?`.

Reply:
But for what purpose?

Try to search for `[NotNullWhen(true)]` occurrences and you can see that it's almost always accompanied with `?`. Also it does not make sense to allocate memory when it's not needed.